Homework2/ICurrentValueCalc.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Homework2
+{
+    public interface ICurrentValueCalc
+    {
+        int Nominal { get; set; }
+        double PriceTrade { get; set; }
+        double PriceOrigin { get; set; }
+        string TransType { get; set; }
```

Comment:
If possible we should avoid abbrivaitions. I thik TransactionType name is better